oap-stdlib/src/test/java/oap/application/ApplicationConfigurationTest.java

```diff
@@ -70,14 +70,13 @@ public void testProfiles() {
         assertThat( ac.getProfiles() ).isEmpty();
         ac.reset();
 
-        ac.profiles.add( "-with-file" );
         ac.profiles.add( "test" );
```

Comment:
Could you add `assertThat` after `ac.profiles.add..`?

Reply:
Yes, tried to add. It behaves the same as in master if to add the same line after add.profiles.
The test start to show wrong results in comparison